API/Configurations/OsuConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Configurations
+{
+    public class OsuConfiguration
+    {
+        public const string Position = "Osu";
+
+        [Required(ErrorMessage = "ApiKey is required!")]
+        public string ApiKey { get; set; } = string.Empty;
+
+        [Required(ErrorMessage = "ClientId is required!")]
+        [Range(1, long.MaxValue, ErrorMessage = "ClientId must be a positive, non-zero value!")]
+        public long ClientId { get; set; } = 0;
+
+        [Required(ErrorMessage = "ClientSecret is required!")]
+        public string ClientSecret { get; set; } = string.Empty;
+        public bool AutoUpdateUsers { get; set; } = false;
+        public bool AllowDataFetching { get; set; } = false;
```

Comment:
bools are false by default right?

Reply:
Oh I actually didn't know this, you're right they are false by default